Makefile

```diff
@@ -283,12 +281,13 @@ lint:
 
 # This is safe; recommended even.
 dep: vendor
-vendor: Gopkg.*
-	dep ensure --vendor-only
+vendor: go.mod go.sum
+	go mod vendor
```

Comment:
With Go modules, `go mod vendor` is for populating a `vendor/` dir for checking into your repo. Since we don't bundle vendor deps here, we can simply do `go go mod download` before build. But that's optional, and usually just done as a pre-validation CI step. The `vendor` step isn't necessary when building, as `go build` will auto-download deps.

Reply:
I'm just using this as a pre-validation CI step. Still learning modules and will check out `mod download` - seems better.

Reply:
Please leave all the feedback you'd like!! Thanks!